packages/grafana-ui/src/components/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -46,7 +46,7 @@ export class ErrorBoundary extends PureComponent<Props, State> {
   }
 }
 
-interface WithAlertBoxProps {
+export interface WithAlertBoxProps {
```

Comment:
Probably need to add some docs here if we are going to export it.

Reply:
done